trackio/deploy.py

```diff
@@ -75,12 +105,14 @@ def deploy_as_space(
     )
 
     huggingface_hub.utils.disable_progress_bars()
-    hf_api.upload_folder(
-        repo_id=space_id,
-        repo_type="space",
-        folder_path=trackio_path,
-        ignore_patterns=["README.md"],
-    )
+
+    if is_source_install:
+        hf_api.upload_folder(
```

Comment:
Without the upload_folder, the Space repo won't contain a copy of trackio, but it will be in requirements.txt. How will the Space know to launch ui.py from the installed trackio package as a Gradio app?